roles/tobiko/tasks/main.yml

```
@@ -33,6 +33,29 @@
     tasks_from: create-clouds-file
   when: not cifmw_tobiko_dry_run | bool
 
+
+- name: Generate ssh keys used for the VMs that tobiko creates
+  block:
+    - name: Check if the ssh keys already exist
+      ansible.builtin.stat:
+        path: "{{ cifmw_tobiko_artifacts_basedir }}/id_ecdsa"
+      register: check_id_ecdsa
+
+    - name: Create the ssh keys
+      community.crypto.openssh_keypair:
+        path: "{{ cifmw_tobiko_artifacts_basedir }}/id_ecdsa"
+        type: ecdsa
+        passphrase: ''
+      when: not check_id_ecdsa.stat.exists
+
+- name: Create tobiko.conf
+  vars:
+    tobikoconf_combined: "{{ cifmw_tobiko_tobikoconf_default | combine(cifmw_tobiko_tobikoconf ,recursive=True) }}"
```

So that we avoid a long line (also, note the space location in the `combine` call)
```suggestion
    tobikoconf_combined: >-
      {{
          cifmw_tobiko_tobikoconf_default |
          combine(cifmw_tobiko_tobikoconf, recursive=True)
      }}
```